components/match_table/commons/match_table.lua

```diff
@@ -417,8 +417,8 @@ end
 ---@return {index: number, link: string}[]
 function MatchTable:vodsFromRecord(record)
 	local vods = {}
-	if String.nilIfEmpty(record.vod) then
-		vods = {{index = 0, link = record.vod}}
+	if String.nilIfEmpty(record.links.vod) then
```

Comment:
links is not retrieved, you need to adjust the query param in the query calls too (lines 290-291)

mind that needs to be done in sc2 infobox player too